packages/react-composites/src/composites/CallWithChatComposite/CallWithChatPane.tsx

```diff
@@ -87,6 +90,7 @@ export const CallWithChatPane = (props: {
     </CallAdapterProvider>
   );
 
+  const modalLayerHost = document.getElementById(props.modalLayerHostId);
```

Comment:
Can we instead use callWIthChatCompositeRootDiv.getElementById() instead of document?
If Contoso ever has more than one composite on the page this will break.
Don recently added a reference to this rootDiv to do the same thing

Reply:
I tested this with two composites side by side and it works fine. 
https://user-images.githubusercontent.com/79475487/161810214-6cdb126f-2042-4cc0-94a8-940c50e4d316.mp4
This is because the id of the modal layer host is uniquely generated in the CallWithChat composite with `useId` from '@fluentui/react-hooks'
I could change reference the CallWithChat composite div but it would be another prop that needs to be added to CallWIthChatPane.

Reply:
Oh nice! Nope what you have sounds good!